contract/test/test-contract.js

```diff
@@ -59,7 +69,7 @@ test('Install the contract', async t => {
 });
 
 test('Start the contract', async t => {
-  const { zoe, bundle } = t.context;
+  const { zoe, bundle } = /** @type {TestContext} */ (t.context);
```

Comment:
now that `test` is typed, all these casts of `t.context` are redundant.

please remove